Gruntfile.js

```diff
@@ -65,12 +65,12 @@ module.exports = function( grunt ) {
 					cleanup : ['style', 'fill', 'id'],
 				},
 				files: {
-					'svg-sprite/dashicons.svg': ['svg-min/*.svg']
+					'svg-sprite/dashicons.svg': [ 'svg-min/*.svg', 'svg-min/**/*.svg' ]
```

Comment:
Similar to the above, you'd need just: `'svg-sprite/dashicons.svg': [ 'svg-min/**/*.svg' ]`